modules/questions.py

```diff
@@ -63,7 +70,10 @@
 
 
 class Questions(Module):
-    AUTOPOST_QUESTION_INTERVAL = timedelta(hours=6)
+    AUTOPOST_NOT_STARTED_MSG_PREFIX = (
+        "Whoever starts working on this question is gonna get a BIG STAMP from me!"
+    )
+    AUTOPOST_STAGNANT_MSG_PREFIX = "Whoever picks up working on some of these questions is gonna get a BIG STAMP from me!"
```

Comment:
who is "me" here? is stampy going to give stamps? literally the discord reactions?!? wont' that mess up karma calculation? 👀

Reply:
Hm, it was meant to be metaphorical but I guess you're right :sweat_smile: